plugins/rich-editor/src/scripts/components/EditorEmojiPicker.jsx

```diff
@@ -52,6 +56,20 @@ export default class EditorEmojiPicker extends React.Component {
         }
     }
 
+    /**
+     * Close if we lose focus on the component
+     * @param {React.Event} event - A synthetic event.
```

Comment:
This should be `React.SyntheticEvent` or better `React.FocusEvent`.

Also the parameter name doesn't line up with the docblock.